README.md

```diff
@@ -44,9 +44,11 @@ the [rcgen](https://docs.rs/rcgen) crate can be used to create new certificates
 This crate intentionally **does not** implement `Clone` on private key types in
 order to minimize the exposure of private key data in memory.
 
-Since these types are immutable, if you find you're self wanting to clone them
-it may be better to consider wrapping the `PrivateKeyDer<'_>` in a [`Rc`]` or
-[`Arc`].
+If you want to convert a `PrivateKeyDer<'_>` with a lifetime to an owned
```

Comment:
IMO worth mentioning `&PrivateKeyDer` here also

Reply:
Also a `PrivateKeyDer<'static>` technically has a lifetime? Maybe something like "If you want to extend the lifetime of a `PrivateKeyDer<'_>`, consider ..".

Reply:
I tweaked the language.